source/ambassador_host.go

```diff
@@ -164,7 +164,7 @@ func (sc *ambassadorHostSource) Endpoints(ctx context.Context) ([]*endpoint.Endp
 }
 
 // endpointsFromHost extracts the endpoints from a Host object
-func (sc *ambassadorHostSource) endpointsFromHost(ctx context.Context, host *ambassador.Host, targets endpoint.Targets) ([]*endpoint.Endpoint, error) {
+func (sc *ambassadorHostSource) endpointsFromHost(_ context.Context, host *ambassador.Host, targets endpoint.Targets) ([]*endpoint.Endpoint, error) {
```

Comment:
@mloiseleur is this part of an interface that we can't change? Otherwise I would prefer that we don't pass a context that is not used.

Reply:
It seems to work without it. See d293ed24